### PR TITLE
added Russian messages

### PR DESCRIPTION
**Pull request by anton-arsentyev**
doc: added Russian messages

### CODE REVIEW

**Comment by rnveach** [comment]
@anton-arsentyev For full russian support, are you planning to translate all the messages?

https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/com/puppycrawl/tools/checkstyle/messages.properties
https://github.com/checkstyle/checkstyle/tree/master/src/main/resources/com/puppycrawl/tools/checkstyle/checks and the sub-directories

@romani Russian would have to be added to:
https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_checks.xml#L72
https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java#L52-L62

File: src/main/resources/com/puppycrawl/tools/checkstyle/messages_ru.properties [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# for DefaultLogger.java
+#
+DefaultLogger.addException=Ошибка проверки {0}
+DefaultLogger.auditStarted=Начинаем проверку...
+DefaultLogger.auditFinished=Проверка закончена.
+
+PackageObjectFactory.unableToInstantiateExceptionMessage=Невозможно инициализировать ''{0}'' \
+   класс, также невозможно инициализировать его как {1}. \
+   Пожалуйста проверьте что имя класса указанно верно или прочитайте как настроить \
+   использование коротких имен http://checkstyle.sourceforge.net/config.html\#Packages.
```
Comment (rnveach) [inline src/main/resources/com/puppycrawl/tools/checkstyle/messages_ru.properties]: This line looks like it is missing an `\` signifying it spans to the next line.

**Comment by anton-arsentyev** [comment]
@rnveach how many lines to translate approx. ?
I can do it in this PR.

**Comment by rnveach** [comment]
@anton-arsentyev Just estimating on the number of message variables we have, it would be around 259.
I believe all our messages are one liners.

**Comment by anton-arsentyev** [comment]
@rnveach I think its possible, not today ofc. but still ;)
will translate as much as I can

File: src/main/resources/com/puppycrawl/tools/checkstyle/messages_ru.properties [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+#
+# for DefaultLogger.java
+#
+DefaultLogger.addException=Ошибка проверки {0}
+DefaultLogger.auditStarted=Начинаем проверку...
+DefaultLogger.auditFinished=Проверка закончена.
+
+PackageObjectFactory.unableToInstantiateExceptionMessage=Невозможно инициализировать ''{0}'' \
+   класс, также невозможно инициализировать его как {1}. \
+   Пожалуйста проверьте что имя класса указанно верно или прочитайте как настроить \
```
Comment (MEZk) [inline src/main/resources/com/puppycrawl/tools/checkstyle/messages_ru.properties]: 1) There should be a comma after the word "Пожалуйста". 
2) There should be a comma before the word "что".

File: src/main/resources/com/puppycrawl/tools/checkstyle/messages_ru.properties [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+   использование коротких имен http://checkstyle.sourceforge.net/config.html\#Packages. \
+   Пожалуйста также проверьте что ClassLoader для Checker-а настроен правильно.
```
Comment (MEZk) [inline src/main/resources/com/puppycrawl/tools/checkstyle/messages_ru.properties]: 1) There should be a comma after the word "Пожалуйста".
2) There should be a comma before the word "что".

File: src/main/resources/com/puppycrawl/tools/checkstyle/messages_ru.properties (hunk 2 above)
Comment (Vladlis) [inline src/main/resources/com/puppycrawl/tools/checkstyle/messages_ru.properties]: I believe there also should be a comma after "верно". Please take a look at [wikipedia](https://ru.wikipedia.org/wiki/%D0%A1%D0%BB%D0%BE%D0%B6%D0%BD%D0%BE%D0%BF%D0%BE%D0%B4%D1%87%D0%B8%D0%BD%D1%91%D0%BD%D0%BD%D0%BE%D0%B5_%D0%BF%D1%80%D0%B5%D0%B4%D0%BB%D0%BE%D0%B6%D0%B5%D0%BD%D0%B8%D0%B5).

**Comment by rnveach** [comment]
@anton-arsentyev Do you still plan to try and do this?

**Comment by romani** [comment]
If no activity happen till end the month, PR will be closed (changes are too small to keep it).

**Comment by anton-arsentyev** [comment]
Translated half of the messages, dont sure if I will finish all till end of the March

**Comment by romani** [comment]
@anton-arsentyev , please push changes to your branch periodically.
Resolve our points and reply "done" for each raised point.
That will help us to give you feedback continuously and understand that PR is not abandoned.

Thank a lot for your help!

**Comment by romani** [comment]
@anton-arsentyev , please make our CIs happy.

**Comment by rnveach** [comment]
@anton-arsentyev Only test failing is commit naming. It will look at all your commits, not just your last one.
You are better off squashing and having one commit unless you want to rename them all.

**Comment by romani** [comment]
@anton-arsentyev , please amend commit message to be single line 
https://travis-ci.org/checkstyle/checkstyle/jobs/209835902#L1071

wercker failure is not relevant to changes - should be fine on next relaunch.

**Comment by romani** [comment]
@anton-arsentyev , if not progress till May1 we will close this PR.
Sorry but this is required to minimize our time of PR/issues management.

**Comment by anton-arsentyev** [comment]
@romani ok, will try to fix problems

**Comment by romani** [comment]
looks like Russian translation is not in need.
I am closing this PR only to ease our management of PRs, work on issue could continue if smb have a time and desire to help.

**Pull request closed without merging**